Editors/RoomEditor.cpp

```diff
@@ -116,7 +116,8 @@ RoomEditor::RoomEditor(MessageModel* model, QWidget* parent) : BaseEditor(model,
 
   // This updates all the model views in the event of a sprite is changed
   connect(MainWindow::resourceMap, &ResourceModelMap::DataChanged, this, [this]() {
-    _ui->entitiesListView->reset();
+    // _ui->entitiesListView->reset();
+    _ui->elementsListView->reset();
```

Comment:
@RobertBColton There is no ``entitiesListView`` in the ui file, I found ``elementsListView`` instead.